releasenotes/notes/capi-changes-3275e667680af8e1.yaml

```
@@ -0,0 +1,10 @@
+---
+prelude: >
+    Docs, container tags and scripts for magnum capi changes
+features:
+  - |
+    - new environment setup scripts for az-config, os-config + skc
+    - new container tag added for magnum with new driver included
```

```suggestion
features:
  - new environment setup scripts for az-config, os-config + skc
  - new container tag added for magnum with new driver included
```
The formatting for release notes is a bit awkward. This way should present the two lines as two separate items in the list